evm-config.schema.json

```
@@ -143,6 +144,14 @@
         "extends"
       ]
     },
+    {
+      "properties": {
+        "defaultTarget": {
+          "type": "string",
+          "pattern": "chrome"
+        }
+      }
```

```suggestion
      },
      "required": [
        "defaultTarget"
      ]
```

Since `defaultTarget` is normally optional, without requiring it in this `oneOf` schema it will match any schema which is missing `defaultTarget`, which then cause the error of matching multiple `oneOf` schemas.